src/pages/mypage/ui/main/components/MyLikedCdList.tsx

```
@@ -7,16 +8,49 @@ import { useMyLikedCdList } from '@/entities/playlist/model/useMyCd'
 import { CdNameInfo } from '@/pages/mypage/ui/main/components'
 import { useSingleSelect } from '@/shared/lib/useSingleSelect'
 import { flexColCenter } from '@/shared/styles/mixins'
-import { Loading, Error, ContentHeader, Cd } from '@/shared/ui'
+import { Loading, Error, ContentHeader, Modal } from '@/shared/ui'
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)